cms/admin/change_list.py

```diff
@@ -71,10 +71,14 @@ def __init__(self, request, *args, **kwargs):
         except:
             raise
         self.get_results(request)
-        
```

Comment:
Usually it's not a good idea to chage code style or start reformatting code for forked projects. It makes harder to merge new changes from the original repository.